pkg/util/credentials/credentials.go

```diff
@@ -101,6 +102,7 @@ func ProviderCredentials(p kubeone.CloudProviderName) (map[string]string, error)
 			{Name: "OS_PASSWORD"},
 			{Name: "OS_DOMAIN_NAME"},
 			{Name: "OS_REGION_NAME"},
+			{Name: "OS_TENANT_ID"},
```

Comment:
The problem here is that only ether of two (`OS_TENANT_ID` or `OS_TENANT_NAME`) is required, but not both simultaneously. But this line makes both of them always required, which is not what MC expect.

Reply:
Both are not required to be present, but what happens in such case? Should we throw an error or have higher priority for one of them?

Reply:
That's the trick, we need to make sure that one of ether is present, we don't need both of them, [here's MC error check](https://github.com/kubermatic/machine-controller/pull/580/files#diff-e07452860a04f7019cbb8dfb86d357c1R353)